internal/pkg/util/configurebccsp.go

```diff
@@ -59,8 +60,15 @@ func ConfigureBCCSP(optsPtr **factory.FactoryOpts, mspDir, homeDir string) error
 		log.Debugf("Initializing BCCSP with software options %+v", opts.SwOpts)
 	}
 	if opts.Pkcs11Opts != nil {
-		log.Debugf("Initializing BCCSP with PKCS11 options %+v", opts.Pkcs11Opts)
+		log.Debugf("Initializing BCCSP with PKCS11 options %+v", sanitizePkcs11Opts(*opts.Pkcs11Opts))
 	}
 	*optsPtr = opts
 	return nil
 }
+
+// redacts label and pin from PKCS11 opts
+func sanitizePkcs11Opts(opts pkcs11.PKCS11Opts) pkcs11.PKCS11Opts {
+	opts.Pin = strings.Repeat("*", len(opts.Pin))
+	opts.Label = strings.Repeat("*", len(opts.Label))
```

Comment:
It's probably not a big deal but if we're trying to redact the sensitive information, let's not use the length of the fields to determine how many asterisks are used. Just put down 6 or 8 of them and move on.